tests/backup_helper.go

```diff
@@ -2672,8 +2769,7 @@ func ValidateBackup(ctx context1.Context, backupName string, orgID string, sched
 			return "", false, fmt.Errorf("InspectBackup Err: %v", err)
 		}
 		backupStatus := backupInspectResponse.GetBackup().GetStatus().Status
-		if backupStatus == api.BackupInfo_StatusInfo_Success ||
```

Comment:
@mkoppal-px @ak-px please review this change, as I have removed the check for partial success from the common method ValidateBackup().